h2o-mapreduce-generic/src/main/java/water/hadoop/DelegationTokenRefresher.java

```diff
@@ -67,6 +67,7 @@ public DelegationTokenRefresher(
 
   public void start() {
     if (HiveTokenGenerator.isHiveDriverPresent()) {
+      refreshTokens();
```

Comment:
this is unrelated, but improves user experience, since delegation token is acquired immediatelly after cluster startup